docs/components/authentication/sign-in.mdx

```diff
@@ -28,8 +28,10 @@ All props are optional.
 | `routing` | `'hash' \| 'path' \| 'virtual'` | The routing strategy for your pages.<br />Defaults to `'path'` in Next.js and Remix applications. Defaults to `hash` for all other SDK's. |
 | `path` | `string` | The path where the component is mounted on when `routing` is set to `path`. It is ignored in hash- and virtual-based routing.<br />For example: `/sign-in`. |
 | `signUpUrl` | `string` | Full URL or path to the sign up page. Use this property to provide the target of the 'Sign Up' link that's rendered. It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables#sign-in-and-sign-up-redirects) instead. |
-| `signInFallbackRedirectUrl?` | `string` | The fallback URL to redirect to after the user signs in, if there's no `redirect_url` in the path already. Defaults to `/`. It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables#sign-in-and-sign-up-redirects) instead. |
-| `signInForceRedirectUrl?` | `string` | If provided, this URL will always be redirected to after the user signs in. It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables#sign-in-and-sign-up-redirects) instead. |
+| `forceRedirectUrl?` | `string` | If provided, this URL will always be redirected to after the user signs in. Takes priority over legacy props such as `afterSignInUrl` and `redirectUrl`. It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables#sign-in-and-sign-up-redirects) instead. |
```

Comment:
I'd probably order these the following way:
- forceRedirectUrl
- fallbackRedirectUrl
- signUpForceRedirectUrl
- signUpFallbackRedirectUrl

My reasoning is that the first two affect the primary flow of the component (signing-in) and the last two affect the what we call "transfer flow" (signing up through a SignIn component as a result of following an oauth flow). Could this be something we also want to document?